Makefile

```diff
@@ -122,11 +122,11 @@ test-all-sizes-ci:
 prefix=/usr/local
     
 install: all
-	install -m 0755 bin/csvcut $(prefix)/bin/csvcut
-	install -m 0755 bin/csvgrep $(prefix)/bin/csvgrep
-	install -m 0755 bin/csvawk $(prefix)/bin/csvawk
-	install -m 0755 bin/csvpipe $(prefix)/bin/csvpipe
-	install -m 0755 bin/csvunpipe $(prefix)/bin/csvunpipe
+	install -m 0755 bin/csvcut $(DESTDIR)$(prefix)/bin/csvcut
```

Comment:
Is `$DESTDIR` something else than `$prefix`? I intended the prefix variable to use like this: `make install prefix=/usr/local` for example?

Reply:
They are pretty close,  `DESTDIR` allows for all macros and existing automation in packaging tools (.deb, .rpm) to work as is.

Eg in rpmspec, `%make_install` is a macro doing`make install DESTDIR=...`. Of course one could redefine macros, overrides stuff, but mostly all tools will expect `DESTDIR` (as it's also part of GNU make coding standards). 

rpm build is going to do make install DESTDIR=/tmp/tmproot ... then tar every thing in order to have the actual package archive. So there is a difference between the two. You probably intended `prefix` to be more `/usr`, `/usr/local`, while `DESTDIR` is more  expected to be `/` or a temporary folder to build the archive, thus the $(DESTDIR)$(prefix)

So in theory one could do `make install DESTDIR=/tmp/myalternateroot prefix=/usr/local` to override both.

Reply:
cool, learned something new :)